tests/components/apsystems/conftest.py

```diff
@@ -52,6 +52,12 @@ def mock_apsystems() -> Generator[MagicMock]:
             e2=6.0,
             te2=7.0,
         )
+        mock_api.get_alarm_info.return_value = ReturnAlarmInfo(
+            og=Status.normal,
+            isce1=Status.alarm,
+            isce2=Status.normal,
+            oe=Status.alarm,
```

Comment:
These aren't booleans right?

Reply:
Yes, the Apsystems python lib returns an IntEnum `Status` class here: https://github.com/SonnenladenGmbH/APsystems-EZ1-API/blob/main/APsystemsEZ1/__init__.py#L164

Reply:
Do you have a link to the Status intenum?

Reply:
Shure: https://github.com/SonnenladenGmbH/APsystems-EZ1-API/blob/main/APsystemsEZ1/__init__.py#L8

Reply:
I'm not sure why this is an enum and not just a boolean tbf

Reply:
That's right. But that's something only @mawoka-myblock can say something about. 😉

Reply:
(it also wasn't a comment for you, just thinking out loud)

Reply:
Not really sure here, but I got the python api file from a beginner and started improving it from there while sticking as close to the api as possible. So that's how you sometimes miss things like that, right? But I'd say that it's not a huge problem, is it?

Reply:
No it's not, but it would be nice to eventually change this, this way you could remove the `bool` from `binary_sensor.py`. Because when you see an enum you automatically get the idea there are more options, but if its only true or false, a boolean makes more sense.